R/neighborhood_analysis.R

```diff
@@ -282,10 +286,10 @@ RegionNeighbors.Seurat <- function (
       select(-barcode) |>
       setNames(nm = c("var1", "var2")) |>
       mutate_if(is.factor, as.character) |>
-      mutate(var2 = case_when((!is.na(var2)) & (var1 != var2) ~ paste0(column_key, lbl),
+      mutate(var2 = case_when((!is.na(var2)) & (var1 != var2) ~ paste0(column_key, gsub(" ", "\\.", lbl)),
                               TRUE ~ var2)) |>
       select(var2) |>
-      setNames(nm = paste0(column_key, lbl))
+      setNames(nm = paste0(column_key, gsub(" ", "\\.", lbl)))
```

Comment:
Not entirely sure what this does, but aren't the new names already stored in `new_lbl`? Is there a reason to compute these again?

Reply:
Rewrote this entirely to stop function if blank spaces are detected